src/test/java/com/faforever/client/fx/PlayerContextMenuControllerTest.java

```diff
@@ -0,0 +1,132 @@
+package com.faforever.client.fx;
+
+import com.faforever.client.builders.PlayerBeanBuilder;
+import com.faforever.client.config.ClientProperties;
+import com.faforever.client.domain.GameBean;
+import com.faforever.client.domain.PlayerBean;
+import com.faforever.client.player.PlayerService;
+import com.faforever.client.test.UITest;
+import com.faforever.client.theme.UiService;
+import com.google.common.eventbus.EventBus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.testfx.util.WaitForAsyncUtils;
+
+import static com.faforever.client.player.SocialStatus.FOE;
+import static com.faforever.client.player.SocialStatus.FRIEND;
+import static com.faforever.client.player.SocialStatus.OTHER;
+import static com.faforever.client.player.SocialStatus.SELF;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+public class PlayerContextMenuControllerTest extends UITest {
+  private static final String TEST_USER_NAME = "junit";
+
+  @Mock
+  private ClientProperties clientProperties;
+  @Mock
+  private UiService uiService;
+  @Mock
+  private PlayerService playerService;
+  @Mock
+  private EventBus eventBus;
+  @Mock
+  private PlatformService platformService;
+
+  private PlayerContextMenuController instance;
+  private PlayerBean player;
+
+  @BeforeEach
+  public void setUp() throws Exception {
+    instance = new PlayerContextMenuController(clientProperties, playerService,
+        eventBus, uiService, platformService);
+    loadFxml("theme/player_context_menu.fxml", clazz -> instance);
```

Comment:
Somehow the initialize() method doesn't get called, so I get NullPointerExceptions when calling setPlayer(). I don't understand why it doesn't load the fxml properly though.